exporters/otlp/README.md

```diff
@@ -3,13 +3,146 @@
 [![GoDoc](https://godoc.org/go.opentelemetry.io/otel?status.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp)
 
 
-This exporter converts OpenTelemetry [SpanData](https://github.com/open-telemetry/opentelemetry-go/blob/6769330394f78192df01cb59299e9e0f2e5e977b/sdk/export/trace/trace.go#L49) 
-to OpenTelemetry Protocol [Span](https://github.com/open-telemetry/opentelemetry-proto/blob/c20698d5bb483cf05de1a7c0e134b7c57e359674/opentelemetry/proto/trace/v1/trace.proto#L46)
-and exports them to OpenTelemetry Collector.
+This exporter exports OpenTelemetry spans and metrics to the OpenTelemetry Collector.
 
 
-## Installation
+## Installation and Setup
+
+The exporter can be installed using standard `go` functionality.
 
 ```bash
 $ go get -u go.opentelemetry.io/otel/exporters/otlp
 ```
+
+A new exporter can be created using the `NewExporter` function.
+
+```golang
+package main
+
+import (
+	"log"
+
+	"go.opentelemetry.io/otel/exporters/otlp"
+	"go.opentelemetry.io/otel/sdk/metric/controller/push"
+	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+func main() {
+	exporter, err := otlp.NewExporter() // Configure as needed.
+	if err != nil {
+		log.Fatalf("failed to create exporter: %v", err)
+	}
+	defer func() {
+		err := exporter.Stop()
+		if err != nil {
+			log.Fatalf("failed to stop exporter: %v", err)
+		}
+	}()
+
+	// Note: The exporter can also be used as a Batcher.
```

Comment:
What did you mean, here?

Reply:
I updated the comment to include an example of registering the exporter as a Batcher. That was the intention I had. LMK if that makes sense.

Reply:
:+1: